functions/Measure-DbaDiskSpaceRequirement.ps1

```diff
@@ -70,186 +76,224 @@ Function Measure-DbaDiskSpaceRequirement {
         }
-        if($DB2) { # Files on destination but not on source (strange scenario but possible)
-            $DataFiles3 = Compare-Object -ReferenceObject $DataFiles2 -DifferenceObject $DataFiles1 -Property Name -PassThru
-            foreach($File3 in $DataFiles3) {
-                $Detail += @([PSCustomObject]@{
-                    Source = $Source
-                    Destination = $Destination
-                    DatabaseName1 = $SourceDatabase
-                    DatabaseName2 = $DB2.Name
-                    Name1 = $NullText
-                    Name2 = $File3.Name
-                    FilePath1 = $NullText
-                    FilePath2 = $File2.FileName
-                    SizeKB1 = 0
-                    SizeKB2 = $File3.Size * -1
-                    DiffKB = $File3.Size * -1
-                    ComputerName = $ComputerName
-                    MountPoint = Get-MountPointFromPath -Path $File3.Filename -ComputerName $ComputerName -Credential $Credential
-                })
+        if ($destDb) {
+            # Files on destination but not on source (strange scenario but possible)
+            $destFilesNotSource = Compare-Object -ReferenceObject $destFiles -DifferenceObject $sourceFiles -Property Name -PassThru
+            foreach ($destFileNotSource in $destFilesNotSource) {
+                $details += @([PSCustomObject]@{
+                    SourceComputerName      = $sourceServer.NetName
+                    SourceInstance          = $sourceServer.ServiceName
+                    SourceSqlInstance       = $sourceServer.DomainInstanceName
+                    DestinationComputerName = $destServer.NetName
+                    DestinationInstance     = $destServer.ServiceName
+                    DestinationSqlInstance  = $destServer.DomainInstanceName
+                    SourceDatabaseName = $SourceDatabase
+                        DestinationDatabaseName = $destDb.Name
+                        SourceLogicalName         = $NullText
+                        DestinationLogicalName         = $destFileNotSource.Name
+                        SourceFileName     = $NullText
+                        DestinationFileName     = $destFile.FileName
+                        SourceFileSizeKB       = 0
+                        DestinationFileSizeKB       = $destFileNotSource.Size * -1
+                        DiffKB        = $destFileNotSource.Size * -1
+                        MountPoint    = Get-MountPointFromPath -Path $destFileNotSource.Filename -ComputerName $computerName -Credential $Credential
+                    })
             }
         }
         $DestinationDatabase = ''
     }
     end {
-        $Detail | Select-DefaultView -Property DatabaseName1, DatabaseName2, Name1, Name2, SizeKB1, SizeKB2, DiffKB, ComputerName, MountPoint
+        if (Test-FunctionInterrupt) { return }
+
+        $details | Select-DefaultView -Property DatabaseName1, DatabaseName2, Name1, Name2, SizeKB1, SizeKB2, DiffKB, ComputerName, MountPoint
```

Comment:
Unfortunately, I didn't see much in the output for this PR. Something went wrong somewhere.